enhancements/kube-apiserver/static-pods-for-single-node-via-ignition.md

```diff
@@ -0,0 +1,92 @@
+---
+title: static-pods-for-single-node-via-ignition
+authors:
+  - "@deads2k"
+reviewers:
+approvers:
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also:
+  - https://github.com/sjenning/rhcos-kaio  
+replaces:
+superseded-by:
+---
+
+# Static Pods for Single Node Cluster via Ignition
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+1. Do we even have a use-case for spending the time building this thought experiment?
```

Comment:
@mrguitar @imcleod do you have any?

Reply:
The ILT is working on collecting use cases for single node clusters. Progress has been slow on my side with other commitments but we'll have this put together soon. In the absence of this, people keep wanting to take a single physical server and create a 3 node cluster w/ three VMs on the single node. The overhead of that is insane IMO. Regardless, single node clusters are on the roadmap so please spend the cycles on this.